test/bitcoin/bitcoinTest.mo

```diff
@@ -0,0 +1,82 @@
+import Transaction "../../src/bitcoin/Transaction";
+import Script "../../src/bitcoin/Script";
+import Types "../../src/bitcoin/Types";
+import BitcoinTestTools "./bitcoinTestTools";
+import Blob "mo:base/Blob";
+import Debug "mo:base/Debug";
+
+let TxIn = Transaction.TxIn;
+let TxOut = Transaction.TxOut;
+let Account = BitcoinTestTools.Account;
+let ecdsaSign = BitcoinTestTools.ecdsaSign;
+let signatureToDer = BitcoinTestTools.signatureToDer;
+
+let wallet = {
+  account1 = Account(22265090479312778178772228083027296664144);
+  account2 = Account(29595381593786747354608258168471648998894101022644411052850960746671046944116);
+  account3 = Account(29595381593786747354608258168471648998894101022644411057647114205835530364276);
+};
+
+do {
+  // Create Transaction inputs.
+  let txIn1 = TxIn({
+    txid = Blob.fromArray([
```

Comment:
Where do these bytes come from?

Reply:
This is just the hash of the transaction which this tx input is using its output. And those specific bytes are just test values I obtained so that I can check that the final signed transaction value is correct.